src/libs/NetworkConnection.js

```diff
@@ -119,6 +119,13 @@ function onReconnect(callback) {
     return () => delete reconnectionCallbacks[currentID];
 }
 
+/**
+ * Delete all queued reconnection callbacks
+ */
+function clearReconnectionCallbacks() {
+    _.each(_.keys(reconnectionCallbacks), key => delete reconnectionCallbacks[key]);
```

Comment:
NAB We could also change `const reconnectionCallbacks = {}` to `let reconnectionCallbacks = {}` and then assign it to an empty object here.

```suggestion
    reconnectionCallbacks = {};
```